.eslintrc

```diff
@@ -2,13 +2,13 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
-    "no-param-reassign": "off",
-    "jsx-boolean-value": "off",
-    "no-use-before-define": "off",
-    "object-shorthand": "off",
+    "no-param-reassign": 0,
```

Comment:
Reasoning for this these changes? These are equivalent, unless there is a an argument for it I'd rather the more meaningful approach.